src/app/components/layouts/column-container/terra-two-columns-container.component.ts

```diff
@@ -37,11 +54,55 @@ export class TerraTwoColumnsContainerComponent
         this.rightColumn = this.leftRightColXS() + this.rightColMD() + this.rightColLG();
     }
 
-    constructor()
+    constructor(private route:ActivatedRoute,
+                private router:Router)
     {
+        this.leftColumn = 'hidden-xs ' + this.leftColMD() + this.leftColLG();
+        this.rightColumn = this.leftRightColXS() + this.rightColMD() + this.rightColLG();
+
+        let subscribable:Observable<Event> = this.router.events.filter((event:RouterEvent) =>
+        {
+            return event instanceof NavigationEnd;
+        });
+
+        this.subscription = subscribable.subscribe((event:NavigationEnd) =>
```

Comment:
Is this necessary in the constructor or can it be moved to `ngOnInit()`? Please keep the constructor as short as possible.